areas-of-d-usage.dd

```diff
@@ -322,4 +322,7 @@ Macros:
                 margin-top: -200px;
             }
         }
+        h2 {
```

Comment:
No, there's little reason to move it to a separate .css file in this case. Inlining things that are only used on one page is the way to go.